app/scripts/lib/channels/web.js

```diff
@@ -22,32 +40,35 @@ define(function (require, exports, module) {
     this._id = id;
   }
 
+  _.extend(WebChannel, COMMANDS);
```

Comment:
Is it worth adding some unit tests to make sure these are present on `WebChannel` or is that overkill because existing tests cover that the commands are all emitted correctly?